src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/state/attributesAccordionActions.ts

```diff
@@ -26,142 +37,95 @@ export const fetchAttributes: ActionCreator<
-  'custom-download/set-orthologue-attributes',
-  (resolve) => {
-    return (orthologueAttributes: AttributesSection) =>
-      resolve(orthologueAttributes);
-  }
-);
+export const setOrthologueAttributes = createStandardAction(
+  'custom-download/set-orthologue-attributes'
+)<AttributeWithOptions>();
 
-export const setOrthologueShowBestMatches = createAction(
-  'custom-download/set-orthologue-show-best-matches',
-  (resolve) => {
-    return (showBestMatches: boolean) => resolve(showBestMatches);
-  }
-);
+export const setOrthologueShowBestMatches = createStandardAction(
+  'custom-download/set-orthologue-show-best-matches'
+)<boolean>();
 
-export const setOrthologueShowAll = createAction(
-  'custom-download/set-orthologue-show-all',
-  (resolve) => {
-    return (showAll: boolean) => resolve(showAll);
-  }
-);
+export const setOrthologueShowAll = createStandardAction(
+  'custom-download/set-orthologue-show-all'
+)<boolean>();
 
-export const setOrthologueApplyToAllSpecies = createAction(
-  'custom-download/set-orthologue-apply-to-all-species',
-  (resolve) => {
-    return (applyToAllSpecies: boolean) => resolve(applyToAllSpecies);
-  }
-);
+export const setOrthologueApplyToAllSpecies = createStandardAction(
+  'custom-download/set-orthologue-apply-to-all-species'
+)<boolean>();
 
-export const setOrthologueSearchTerm = createAction(
-  'custom-download/set-orthologue-search-term',
-  (resolve) => {
-    return (searchTerm: string) => resolve(searchTerm);
-  }
-);
+export const setOrthologueSearchTerm = createStandardAction(
+  'custom-download/set-orthologue-search-term'
+)<string>();
 
 export const setOrthologueSpecies = createAsyncAction(
   'custom-download/set-orthologue-species-request',
   'custom-download/set-orthologue-species-success',
   'custom-download/set-orthologue-species-failure'
-)<{ searchTerm: string }, {}, Error>();
+)<{ searchTerm: string }, CheckboxGridOption[], Error>();
 
 export const fetchOrthologueSpecies: ActionCreator<
-  ThunkAction<void, any, null, Action<string>>
-> = (searchTerm: string, orthologueSpecies: any) => async (dispatch) => {
+  ThunkAction<void, CheckboxGridOption[], null, Action<string>>
+> = (searchTerm: string, orthologueSpecies: CheckboxGridOption[]) => async (
+  dispatch
+) => {
   dispatch(setOrthologueSpecies.request({ searchTerm: searchTerm }));
   try {
     // This will be fetched from the API when we have one
-    let allSpecies = sampleOrthologueSpecies.species;
+    let allSpecies = sampleOrthologueSpecies.species as Species[];
 
-    let filteredSpecies: any = {};
+    let filteredSpecies: CheckboxGridOption[] = [...orthologueSpecies].filter(
+      (species: CheckboxGridOption) => {
+        return species.isChecked;
+      }
+    );
 
-    allSpecies.forEach((species: any) => {
+    allSpecies.forEach((species: Species) => {
       if (
+        searchTerm &&
         species.display_name.toLowerCase().indexOf(searchTerm.toLowerCase()) !==
-        -1
+          -1
```

Comment:
let’s change indexOf !== -1 to either `includes` (with polyfill added in the polyfill.io link in the template file) or regex?